### PR TITLE
Fixed spack package syntax

### PR DESCRIPTION
**Pull request by ravil-mobile**

### CODE REVIEW

File: spack/packages/seissol-utils/package.py [hunk 1]
```diff
@@ -139,24 +139,27 @@ def install(self, spec, prefix):
...
+        bins = [self.prefix.gmsh2gambit,
+                self.prefix.cube_c,
+                self.prefix.rconv,
+                pumgen.prefix.bin,
+                gmsh.prefix.bin]
 
         if "+paraview" in self.spec:
-            bins.append(dependencies["paraview"].spec.previx.bin)
+            paraview = self.spec.dependencies(name='paraview')[0]
+            bins.append(paraview.previx.bin)
```
Comment (Thomas-Ulrich) [inline spack/packages/seissol-utils/package.py]: typo

**Review by Thomas-Ulrich — APPROVED** [review]
(edit) I now unterstand the spec objects.

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
```git='https://github.com/daisy20170101/SeisSol_Cookbook',```
does not exist any more.
(now https://github.com/SeisSol/Examples/ I guess)

File: spack/packages/seissol-env/package.py [hunk 2]
```diff
@@ -102,5 +102,7 @@ def setup_run_environment(self, env):
         env.prepend_path('PYTHONPATH', ":".join(filter(None, pythonpath)))
         
         # add pspamm while loading seissol-env
-        env.prepend_path('PYTHONPATH', self.spec.dependencies_dict()['pspamm'].spec.prefix.pspamm)
-        env.prepend_path('PATH', self.spec.dependencies_dict()['pspamm'].spec.prefix.pspamm)
+        pspamm = self.spec.dependencies(name='py-pspamm')[0]
```
Comment (Thomas-Ulrich) [inline spack/packages/seissol-env/package.py]: You can also write 
```
pspamm = self.spec['py-pspamm']
```
https://spack.readthedocs.io/en/latest/packaging_guide.html#accessing-dependencies

File: spack/packages/seissol-utils/package.py [hunk 3]
```diff
@@ -139,24 +139,27 @@ def install(self, spec, prefix):
...
     
     def setup_run_environment(self, env):
-        dependencies = self.spec.dependencies_dict()
-        bins = [self.spec.prefix.gmsh2gambit, 
-                self.spec.prefix.cube_c, 
-                self.spec.prefix.rconv,
-                dependencies["pumgen"].spec.prefix.bin,
-                dependencies["gmsh"].spec.prefix.bin]
+        pumgen = self.spec.dependencies(name='pumgen')[0]
+        gmsh = self.spec.dependencies(name='gmsh')[0]
```
Comment (Thomas-Ulrich) [inline spack/packages/seissol-utils/package.py]:
```
pumgen = self.spec[pumgen']
gmsh = self.spec[gmsh']
```

File: spack/packages/seissol-utils/package.py [hunk 4]
```diff
@@ -139,24 +139,27 @@ def install(self, spec, prefix):
...
+        bins = [self.prefix.gmsh2gambit,
+                self.prefix.cube_c,
+                self.prefix.rconv,
+                pumgen.prefix.bin,
+                gmsh.prefix.bin]
 
         if "+paraview" in self.spec:
-            bins.append(dependencies["paraview"].spec.previx.bin)
+            paraview = self.spec.dependencies(name='paraview')[0]
```
Comment (Thomas-Ulrich) [inline spack/packages/seissol-utils/package.py]: paraview = self.spec[paraview']

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
Small changes

**Comment by krenzland** [comment]
@ravil-mobile What's the status of this PR?

**Comment by ravil-mobile** [comment]
Hi @krenzland, I honestly cannot remember the current status. I guess I need to make changes suggested by Thomas and test it.

**Comment by Thomas-Ulrich** [comment]
Hi, I've done the proposed changes. Maybe @ravil-mobile can validate and we can merge?

**Comment by krenzland** [comment]
Thanks!

**Pull request merged**